Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1442,7 +1467,9 @@ void TextEditor::insert_at_cursor_or_replace_selection(StringView text)
         && clear_length > 0
         && current_line().leading_spaces() == clear_length;
 
-    execute<InsertTextCommand>(text, m_cursor);
+    auto const no_execute = text == "\t" && has_selection();
+    if (!no_execute)
```

Comment:
Double negative is a code smell

Reply:
Changed. For some reason I had serious brain fog coming up with the case in which we execute which is why I did the double negative, but good catch. Should be fixed now :^)